Bottom: 80fd0b0f8660c8336c295cbc2a24c2b44bdc7f73
Top:    a65b59870a760baf3d148a782f36fe8b306cc71d
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-09-19 12:52:54 -0500

Refresh of add-haproxy-role

---
diff --git a/roles/docker-svn/tasks/main.yml b/roles/docker-svn/tasks/main.yml
index fea7d1b67..7670085f4 100644
--- a/roles/docker-svn/tasks/main.yml
+++ b/roles/docker-svn/tasks/main.yml
@@ -53,11 +53,11 @@
 
 # *****************************************************************************
 # Update or make the image.
-# - name: Checkout image repo
-#   git:
-#     repo: '{{ svn_image_repo }}'
-#     version: master
-#     dest: '{{ docker_projects_dir }}/docker-svn'
+- name: Checkout image repo
+  git:
+    repo: '{{ svn_image_repo }}'
+    version: master
+    dest: '{{ docker_projects_dir }}/docker-svn'
 
 - name: build image
   docker_image:
